src/xdocs/config_header.xml

```diff
@@ -264,14 +263,21 @@ line 6: ^\W*$
         <p>
           Lines 1 and 2 leave room for technical header lines, e.g. the
           "#!/bin/sh" line in Unix shell scripts, or the XML file header
-          of XML files.  Set the multiline property to "1, 2" so these
+          of XML files. Set the multiline property to "1, 2" so these
           lines can be ignored for file types where they do no apply.
           Lines 3 through 6 define the actual header content. Note how
           lines 2, 4 and 5 use escapes for characters that have special
           regexp semantics.
         </p>
       </subsection>
 
+      <subsection name="Notes" id="RegexpHeader_Notes">
```

Comment:
This does not have attention like in other PR, so why is this split?

Reply:
reverted back to one section.